twake/frontend/src/app/scenes/Client/ChannelsBar/ChannelsWorkspace/WorkspaceChannels.tsx

```diff
@@ -1,16 +1,19 @@
 import React from 'react';
 
+import { ChannelResource, ChannelType } from 'app/models/Channel';
+
 import Languages from 'services/languages/languages.js';
+import ModalManager from 'services/Modal/ModalManager';
+import { Collection } from 'services/CollectionsReact/Collections';
+
 import ChannelCategory from '../Parts/Channel/ChannelCategory';
-import ChannelWorkspaceEditor from 'app/scenes/Client/ChannelsBar/Modals/ChannelWorkspaceEditor';
-import ModalManager from 'app/services/Modal/ModalManager';
-import ChannelMenu from '../Parts/Channel/ChannelMenu';
-import WorkspaceChannelList from 'app/scenes/Client/ChannelsBar/Modals/WorkspaceChannelList';
-import ChannelUI from '../Parts/Channel/Channel';
-import { ChannelResource, ChannelType } from 'app/models/Channel';
+import ChannelInter from '../Parts/Channel/channelIntre';
+
+import ChannelWorkspaceEditor from 'scenes/Client/ChannelsBar/Modals/ChannelWorkspaceEditor';
+import WorkspaceChannelList from 'scenes/Client/ChannelsBar/Modals/WorkspaceChannelList';
+
 import Menu from 'components/Menus/Menu.js';
-import Icon from 'app/components/Icon/Icon';
-import { Collection } from 'app/services/CollectionsReact/Collections';
+import Icon from 'components/Icon/Icon';
 
```

Comment:
I can't see it from Github, but please ensure that there is no unused imports :)

Reply:
Done